syntax = "proto3";

message MoveCommand { // mm/s
    int32 speed = 1;
}